src/LabelTrack.cpp

```diff
@@ -55,6 +55,11 @@ static ProjectFileIORegistry::ObjectReaderEntry readerEntry{
    LabelTrack::New
 };
 
+wxString LabelTrack::GetDefaultName()
+{
+   return _("Labels");
```

Comment:
Please see the struct Track::ClassTypeInfo and learn a bit about how it is used.

Maybe it should be extended with a field containing the default name for a track type, so there is a uniform way to specify it for Label and Wave and other types of tracks.

If for some reason there is need for polymorphism in getting "the default name for a track type" but unspecified which type that is.

But this generalization is for now a solution looking for a problem.  Don't do it, but think a little about it.